aidants_connect_web/templates/aidants_connect_web/espace_aidant/home.html

```diff
@@ -94,6 +93,13 @@ <h3>Vos services</h3>
             </a>
           </div>
         {% endif %}
+
+        <div class="fr-col-12 fr-col-md-6 fr-col-lg-4">
+          <a href="{{ sandbox_url }}" class="tile">
+            <span aria-hidden="true"><img class="icon" src="{% static "images/icons/sandbox-icon.png" %}"><br/></span>
```

Comment:
Même si on est dans un aria-hidden, il faut quand même prévoir le alt vide pour les images de décoration. :)

```suggestion
            <span aria-hidden="true"><img alt="" class="icon" src="{% static "images/icons/sandbox-icon.png" %}"><br/></span>
```